server/services/datacommons.py

```diff
@@ -263,6 +263,55 @@ def v2node(nodes, prop):
   })
 
 
+def _merge_paged_response(result, paged_response):
+  for dcid in paged_response.get('data', []):
+    # Initialize dcid in data even when no arcs or properties are returned
+    result.setdefault('data', {}).setdefault(dcid, {})
```

Comment:
nit: maybe use a variable to hold this so can just reuse it in the rest of the code & maybe break apart the setdefault from extend so that it's a little more readable. something like:

```
node_result = result.setdefault('data', {}).setdefault(dcid, {})
....
prop_nodes = node_result.setdefault('arcs', {}).setdefault(prop, {}).setdefault('nodes', [])
prop_nodes.extend(...)
...

Reply:
added, thanks for the tip!